examples/storm-elasticsearch-examples/src/main/java/org/apache/storm/elasticsearch/trident/TridentEsTopology.java

```diff
@@ -38,12 +43,32 @@
 import org.apache.storm.tuple.Fields;
 import org.apache.storm.tuple.Values;
 
-public class TridentEsTopology {
-
-    static final String TOPOLOGY_NAME = "elasticsearch-test-topology2";
+/**
+ * A Trident topology example.
+ * @author unknown
+ */
+public final class TridentEsTopology {
+    /**
+     * The default batch size. Necessary in order to avoid magic number
```

Comment:
Nit: I don't think you need to mention why the field is here. Won't checkstyle complain if someone replaces the field with a magic number again?